test/logger_test.jl

```diff
@@ -0,0 +1,40 @@
+using Test
+using DataFlowTasks
+using DataFlowTasks: R, W, RW
+using LinearAlgebra
+
+computing(mat) = mat*mat
+function work(A, B)
+    @dspawn computing(@RW(A)) label="A"
+    @dspawn computing(@RW(B)) label="B"
+    @dspawn computing(@RW(A)) label="A"
+    @dspawn computing(@RW(B)) label="B"
+    @dspawn computing(@RW(A)) label="A"
+    @dspawn computing(@RW(B)) label="B"
+    DataFlowTasks.sync()
+end
+
+DataFlowTasks.enable_log()
+
+A = ones(20, 20)
+B = ones(20, 20)
+
+work(copy(A), copy(B))
+
+DataFlowTasks.resetlogger!()
+DataFlowTasks.TASKCOUNTER[] = 0
+
+GC.gc()
```

Comment:
Indeed, i'm removing the work on copy. I'll let the reset calls, in case DataFlowTasks was called before this test (don't really know if it can happen in the CI context though).